package.json

```diff
@@ -132,6 +130,10 @@
   "engines": {
     "node": "18.* || 20.* || >= 21"
   },
+  "volta": {
```

Comment:
Can you open a separate PR for this?

Reply:
https://github.com/ember-cli/eslint-plugin-ember/pull/2037